libbackscrub.cc

```diff
@@ -0,0 +1,272 @@
+/* This is licenced software, @see LICENSE file.
```

Comment:
Further split this file into the TFlite parts and the actual mask gen parts.

Reply:
Yep - one source file => one obvious job.